### PR TITLE
Sscssi 270 add review information requested

### PR DESCRIPTION
**Pull request by nixonthapa**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/SSCSSI-270

### Change description ###
This PR adds Review Information Requested, test were successfully ran on both demo and att enviroment.

AAT screenshots
Screenshot 2024-07-15 at 11 09 13

Screenshot 2024-07-15 at 11 18 39

DEMO screenshots
Screenshot 2024-07-15 at 10 03 57

Screenshot 2024-07-15 at 10 10 29

Preview screenshots
tested on the preview env tests are passing

image

### Checklist

- [ x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: functional-test/e2e/request.info.from.party.spec.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+test.describe.serial('WA - Review Information Requested CTSC task initiation and completion tests', {
+    tag: '@work-allocation'
+}, async() => {
+
+    let caseId : string;
+
+    test.beforeAll("Create case and complete request info from party event", async ({requestInfoFromPartySteps}) => {
+        caseId = await createCaseBasedOnCaseType('PIP');
+        await requestInfoFromPartySteps.performRequestInfoFromPartyEvent(caseId);
```
Comment (pavihmcts) [inline functional-test/e2e/request.info.from.party.spec.ts:13]: Can you move this line from beforeAll block to first test? We cannot call the fixtures into the beforeAll block.

`await requestInfoFromPartySteps.performRequestInfoFromPartyEvent(caseId);`

File: functional-test/e2e/request.info.from.party.spec.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+});
+
+test.describe('WA - Review Information Requested CTSC task automatic cancellation when case is void', {
+    tag: '@work-allocation'
+}, async() => {
+
+    let caseId : string;
+
+    test.beforeAll("Case has to be Created", async ({requestInfoFromPartySteps}) => {
+        caseId = await createCaseBasedOnCaseType('PIPINCOMPLETE');
```
Comment (pavihmcts) [inline functional-test/e2e/request.info.from.party.spec.ts:49]: Change the case type to PIP

File: functional-test/e2e/request.info.from.party.spec.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        await requestInfoFromPartySteps.performRequestInfoFromPartyEvent(caseId);
```
Comment (pavihmcts) [inline functional-test/e2e/request.info.from.party.spec.ts:50]: Can you move this line from beforeAll block to first test? We cannot call the fixtures into the beforeAll block.

await requestInfoFromPartySteps.performRequestInfoFromPartyEvent(caseId);

File: functional-test/e2e/request.info.from.party.spec.ts (hunk 1 above)
Comment (nixonthapa) [reply functional-test/e2e/request.info.from.party.spec.ts]: done

File: functional-test/e2e/request.info.from.party.spec.ts (hunk 2 above)
Comment (nixonthapa) [reply functional-test/e2e/request.info.from.party.spec.ts]: done

File: functional-test/e2e/request.info.from.party.spec.ts (hunk 3 above)
Comment (nixonthapa) [reply functional-test/e2e/request.info.from.party.spec.ts]: done

**Review by gokul-sol — APPROVED** [review]

**Review by pavihmcts — APPROVED** [review]

**Pull request merged**